ditto/models/winding.py

```diff
@@ -30,6 +30,9 @@ class Winding(DiTToHasTraits):
     nominal_voltage = Float(
         help="""The nominal voltage of the transformer winding""", default_value=None
     )
+    base_voltage = Float(
+        help="""The voltage basis for per unit calculations""", default_value=None
+    )
```

Comment:
in most cases I believe yes; however, the reason that I added it was that I came across a case where:
```
TRANSFORMER_SETTING.PrimaryBaseVoltage != TRANSFORMER.KVLLprim
```
(where I am using CYME names from the .txt files). 

It's not clear why these two value are different (67 kV vs. 69 kV) but I chose to handle it by setting the actual kV value in openDSS to the KVLLprim and add the PrimaryBaseVoltage to the set of base voltages. This might not be correct.

Reply:
I think that we can eliminate the `base_voltage` field. But I use it in the CYME reader to account for the two different voltage specifications: 
1. KVLLprim/sec (from TRANSFORMER_SETTING)
2. Primary/SecondaryBaseVoltage (from TRANSFORMER).

The BaseVoltage field is used in the opendss writer for three winding transformers; but the KVLL field is used for two winding transformers. I believe that we should be consistent and suggest settling on using the KVLL values from the TRANSFORMER_SETTING. However, I wonder if  there are cases where KVLL is different than BaseVoltage because the BaseVoltage is line to neutral? I think that there are some bus in the openDSS writer right now because of the hard-coded assumptions around voltage being line to line or line to neutral. Perhaps we need to make it more transparent when a voltage value is LL or LN on the Ditto models? Then we can eliminate the need for the hard-coded conversions in the writers.